test/extensions/testData.test.js

```diff
@@ -0,0 +1,29 @@
+const brs = require("brs");
+const { BrsString, BrsInvalid, ValueKind } = brs.types;
+const { Interpreter } = require("../../lib/interpreter");
+const { identifier } = require("../parser/ParserTests");
+
+describe("_brs_.testData", () => {
+    it("check testData object", () => {
+        let _brs_ = new Interpreter().environment.get(identifier("_brs_"));
+        let testData = _brs_.get(new BrsString("testData"));
+
+        expect(testData.kind).toBe(ValueKind.Object);
+        expect(testData.componentName).toBe("roAssociativeArray");
+        expect(testData.elements.size).toBe(0);
+        testData.set(new BrsString("foo"), BrsInvalid.Instance);
+        expect(testData.elements.size).toBe(1);
+
+        testData = new Interpreter().environment
+            .get(identifier("_brs_"))
+            .get(new BrsString("testData"));
+        expect(testData.elements.size).toBe(1);
+
+        brs.resetTestData(); // will clear testData for next new Interpreter instances
```

Comment:
Do we want the `_brs_.testData` to persist across `interpreter` instances?  IIRC there's only one created as part of `roca` anyway.  We'll still want this type of behavior, but perhaps simpler solution would be for each `Interpreter` instance to get its own copy of the `_brs_` extensions 🤔